core/core_test.go

```diff
@@ -236,7 +236,7 @@ func runOrdersyncTestCase(t *testing.T, testCase ordersyncTestCase) func(t *test
 			defer wg.Done()
 			if err := originalNode.Start(); err != nil && err != context.Canceled {
 				// context.Canceled is expected. For any other error, fail the test.
-				require.NoError(t, err)
+				panic(err)
```

Comment:
I ran into this while I was in the middle of refactoring the patch to `getOrdersFromPeer`. There's a good chance we have these in several other places in the codebase.